jobs/build/openshift-online/scripts/merge-and-build-openshift-scripts.sh

```diff
@@ -88,6 +88,20 @@ elif [ "${BUILD_MODE}" == "release" ] ; then
     SPEC_VERSION_COUNT=6
 fi
 
+echo
+echo "=========="
+echo "Setup OIT stuff"
+echo "=========="
+
+
+pushd ${BUILDPATH}
+OIT_DIR="${BUILDPATH}/enterprise-images/"
```

Comment:
I'd prefer WORKSPACE to BUILDPATH. BUILDPATH is a 'go' directory.

Reply:
Again, copy paste from OSE, but will change.